catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitor.java

```diff
@@ -33,17 +38,19 @@
 import ddf.catalog.Constants;
 import ddf.security.common.util.Security;
 
-/**
- * @author rodgersh
- */
+@SuppressWarnings("PackageAccessibility")
```

Comment:
I think this might be specific to IntelliJ Ultimate edition. The IDE keeps flagging OSGi errors without this suppression. Should it stay in the PR?